mvx-esdt-safe/src/register_token.rs

```diff
@@ -245,7 +247,7 @@ pub trait RegisterTokenModule:
     }
 
     #[allow(clippy::field_reassign_with_default)]
-    fn create_event_payment_tuple(
+    fn create_deploy_cost_event_payment_tuple(
```

Comment:
"deploy cost" for what?
```suggestion
    fn create_event_egld_payment_tuple(
```